modules/zetaSspBidAdapter.js

```diff
@@ -156,4 +157,10 @@ function buildBanner(request) {
   };
 }
 
+function provideSharedId(request, payload) {
```

Comment:
since you're intentionally passing all ids, I would rename this method to avoid confusion with the SharedId identity module. maybe `provideSharedIds` or `provideEids` would be better?

Reply:
Done, thanks